src/xdocs/config_filters.xml

```diff
@@ -633,6 +634,161 @@ public class UserService {
       </subsection>
     </section>
 
+    <section name="SuppressionXpathFilter">
```

Comment:
https://github.com/checkstyle/checkstyle/pull/4992#discussion_r137084218
This change must be made.

Reply:
removed